aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -72,9 +79,15 @@ func resourceAwsApiGatewayRestApiCreate(d *schema.ResourceData, meta interface{}
 		description = aws.String(d.Get("description").(string))
 	}
 
+	var policy *string
```

Comment:
This can be simplified with (after `params` declaration):

```go
if v, ok := d.GetOk("policy"); ok && v.(string) != "" {
  params.Policy = aws.String(v.(string))
}
```